### PR TITLE
#338: add basic maintenance page

### PR DESCRIPTION
**Pull request by komejo**
https://github.com/wri/wrin/issues/338

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the main branch (left side). Also you should start your branch off our main.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?
Default maintenance page is busted: 
153503489-954f15a4-2d9a-438d-a37d-e09d521deffc

Issue Number: https://github.com/wri/wrin/issues/338

## What is the new behavior?
New maintenance page is simple, clean: 
Screen Shot 2022-02-21 at 4 45 55 PM

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
This should be deployed prior to the D9 upgrade so any site downtime looks professional.

### CODE REVIEW

File: themes/custom/ts_wrin/templates/ts_custom/maintenance-page.html.twig [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    {% endif %}
+
+    {% if page.content %}
+      {% if is_node %}
+        {{page.content}}
+      {% else %}
+      <div class="landing-page__content container margin-top-md margin-bottom-md">
+        <div class="content__inner" style="text-align: center;">
+          {{ drupal_block('local_tasks_block') }}
+          <img class="logo-color" src="/sites/default/files/nav_logo_color.svg" alt="Home" style="margin: 40px auto;"> {{page.content}}
```
Comment (mariacha) [inline themes/custom/ts_wrin/templates/ts_custom/maintenance-page.html.twig:37]: We can't count on this file existing, or on this page being displayed in English, so can you pull in the branding block? Probably something like `{{ drupal_block('system_branding_block') }}`. Test it out on the China branch to be sure it works. This is what I'm getting here: https://pr-170-wri-china.pantheonsite.io/ with maintenance mode turned on.
Screen Shot 2022-03-10 at 9 22 30 AM

**Review by mariacha — CHANGES_REQUESTED** [review]
This looks great for Flagship, but needs a bit of a tweak for China.

File: themes/custom/ts_wrin/templates/ts_custom/maintenance-page.html.twig (hunk 1 above)
Comment (komejo) [reply themes/custom/ts_wrin/templates/ts_custom/maintenance-page.html.twig]: Fixed and noted the translation points for WRI China in https://github.com/wri/WRIN/issues/338#issuecomment-1064666942

Screen Shot 2022-03-10 at 5 19 53 PM

**Comment by mariacha** [comment]
Very nice!

**Pull request merged**